internal/controllers/upload.go

```diff
@@ -113,6 +132,58 @@ func (c *UploadController) UploadChapterPages(ctx *gin.Context) {
 	ctx.JSON(http.StatusOK, response.SuccessResponse(http.StatusOK, "Files uploaded successfully", uploadResponses))
 }
 
+// ReplacePage handles replace specific page
+func (c *UploadController) ReplacePage(ctx *gin.Context) {
+	mangaID := ctx.Param("manga_id")
+	chapterID := ctx.Param("chapter_id")
+
+	if mangaID == "" || chapterID == "" {
+		ctx.JSON(http.StatusBadRequest, response.ErrorResponse(http.StatusBadRequest, "Manga ID and Chapter ID are required", nil))
+		return
+	}
```

Comment:
chỗ này đang OR mà message and kìa ông <(") với 2 cái này nên tách ra + validate request chỗ khác đi cho controller nó gọn ông